Functions/Office365-Msgtrace/readme.md

```diff
@@ -0,0 +1,57 @@
+---
+page_type: sample
+products:
+- office-365
+- Sentinel
+languages:
+- powershellcore
+extensions:
+  contentType: samples
+  createdDate: 4/2/2020 3:00:56 AM
+description: "This sample can be used to process message traces from an Azure Function."
+---
+
+
+# Ingesting Office 365 Message Traces to Azure Sentinel
+
+Kudos to Stefan Simon! :)
+
+Implementing this script as part of an Azure Function will allow you to ingest Office 365 Message traces to Log Analytics.
+
+### Prerequisites
+
+You need to have an Azure Subscription, ability to create an Azure Function App. You need to have an account with permissions to run get-messagetrace in Office 365.
+Use a dedicated account with a complex pwd stored in Azure Key Vault.
```

Comment:
Is it possible to have a second ARM template that sets up the key vault with the complex password for the service account.